client/signup/style.scss

```diff
@@ -712,6 +712,14 @@ body.is-section-signup.is-white-signup .layout:not( .dops ):not( .is-wccom-oauth
 
 	.signup__step.is-domains,
 	.signup__step.is-emails {
+		@include break-large {
+			margin: 0 0 0 20px;
+			padding-left: 20px;
```

Comment:
We're getting 40px on the left here -- is there a reason you're using both margin and padding here? If we really need the left padding, it may make sense when we shift to `break-wide` below when we move back to `margin: initial`.

Reply:
I've addressed it :) Plus added a few more fixes in domains.